### PR TITLE
Use new status-go and rename NotifyUsers

### PR DESCRIPTION
**Pull request by pedropombeiro**
[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)

### Summary:

[comment]: # (Summarise the problem and how the pull request solves it)

This PR incorporates https://github.com/status-im/status-go/pull/1352

### Testing notes (optional):

Although this is a simple change, it's maybe a good idea to try it out on iOS and one desktop platform just to make sure.

status: ready

### CODE REVIEW

**Comment by pedropombeiro** [comment]
@annadanchenko @asemiankevich is it possible to do a quick test on this PR without waiting for reviews? We need to implement the change from status-go ASAP.

**Comment by asemiankevich** [comment]
sure, no changing fleets? maybe i can run e2e for this too? @PombeirP

**Review by mandrigin — APPROVED** [review]

**Review by rasom — APPROVED** [review]

**Comment by antdanchenko** [comment]
@PombeirP, @asemiankevich all tests are failed against this PR due to super long (2 minutes) account creation, detailed results will be posted in ~10 minutes

**Comment by annadanchenko** [comment]
@PombeirP I can confirm that creating account on iOS and Android takes very long time, so it's not e2e fault but something is wrong with build
full logcat: 
[logcat2.txt](https://github.com/status-im/status-react/files/2772899/logcat2.txt)

logcat contains 
```

01-18 13:22:01.012 15441 15529 E StatusModule: StartNode failed: {"error":"Key: 'NodeConfig.ShhextConfig.BackupDisabledDataDir' Error:Field validation for 'BackupDisabledDataDir' failed on the 'required' tag"}
01-18 13:22:01.013 15441 15529 D ServiceConnector: Sending message to service: 
01-18 13:22:01.013 15441 15441 D StatusService: Received service message.{ when=0 what=0 target=im.status.ethereum.module.StatusService$IncomingHandler }
01-18 13:22:01.078  6929  6929 I PhenotypeExpConfig: refreshConfiguration() : Force = false : UpdateAvailable = false : Age = 189 minutes : MaxAge = 720 minutes
01-18 13:22:01.125 15441 15494 D ReactNativeJS: DEBUG [status-im.utils.handlers:34] - Handling re-frame event:  :show-tab-bar
01-18 13:22:02.061 11343 11360 I Finsky  : [123] com.google.android.finsky.bo.an.run(6): Stats for Executor: VerifyAppsDataStore com.google.android.finsky.bo.ao@b55a25[Running, pool size = 0, active threads = 0, queued tasks = 0, completed tasks = 22]
01-18 13:22:02.073 11343 11360 I Finsky  : [123] com.google.android.finsky.bo.an.run(6): Stats for Executor: Db-verify_apps.db com.google.android.finsky.bo.ao@c4cd7fa[Running, pool size = 0, active threads = 0, queued tasks = 0, completed tasks = 29]
01-18 13:22:03.569 11343 11343 I Finsky  : [2] com.google.android.finsky.scheduler.JobSchedulerEngine$PhoneskyJobSchedulerJobService.onStartJob(8): onJobSchedulerWakeup with jobId 9000
01-18 13:22:03.582 11343 11343 I Finsky  : [2] com.google.android.finsky.scheduler.ad.a(25): Scheduling fallback job with id: 9034, and delay: 43200000 ms
01-18 13:22:03.598 11343 11343 I Finsky  : [2] com.google.android.finsky.scheduler.ad.a(41): Scheduling fallback in 64799993 (absolute: 76839320)
01-18 13:22:03.633 11343 11343 I Finsky  : [2] com.google.android.finsky.scheduler.al.handleMessage(16): DeviceState: DeviceState{currentTime=1547814123623, isCharging=true, isIdle=false, netAny=true, netNotRoaming=true, netUnmetered=true}
01-18 13:22:03.645 11343 15532 I Finsky  : [256] com.google.android.finsky.scheduler.bp.b(89): Jobs in database: 1-1337 12-1 26-1414141414 
01-18 13:22:03.651 11343 11343 I Finsky  : [2] com.google.android.finsky.scheduler.ag.a(59): Running job: 12-1
01-18 13:22:03.652 11343 11343 I Finsky  : [2] com.google.android.finsky.contentsync.ContentSyncJob.a(24): ContentSyncJob started
01-18 13:22:03.656 11343 11343 I Finsky  : [2] com.google.android.finsky.scheduler.ag.a(105): RunningQueue size: 1, PendingQueue size: 0
01-18 13:22:03.658 11343 11343 I Finsky  : [2] com.google.android.finsky.scheduler.ag.a(116): Running queue: 12-1 
01-18 13:22:04.097 11343 11343 I Finsky  : [2] com.google.android.finsky.m.c.a(21): Completed 1 account content syncs with 1 successful.
01-18 13:22:04.107 11343 11343 I Finsky  : [2] com.google.android.finsky.contentsync.ContentSyncJob.a(22): Installation state replication succeeded.
01-18 13:22:04.109 11343 11343 I Finsky  : [2] com.google.android.finsky.scheduler.bm.b(8): jobFinished: 12-1. TimeElapsed: 458ms
01-18 13:22:04.111 11343 11343 I Finsky  : [2] com.google.android.finsky.scheduler.ag.a(139): Job 12-1 finished
01-18 13:22:04.116 11343 11343 I Finsky  : [2] com.google.android.finsky.scheduler.ag.a(105): RunningQueue size: 0, PendingQueue size: 0
01-18 13:22:04.117 11343 11343 I Finsky  : [2] com.google.android.finsky.scheduler.al.handleMessage(42): Executor finished
01-18 13:22:04.161 11343 11343 I Finsky  : [2] com.google.android.finsky.scheduler.bp.b(89): Jobs in database: 1-1337 26-1414141414 
01-18 13:22:04.166 11343 11343 I Finsky  : [2] com.google.android.finsky.scheduler.k.a(69): ConstraintMapping: 1-1337,  -> L: 2955561ms, D: 40924335ms, C: false, I: false, N: 1
01-18 13:22:04.169 11343 11343 I Finsky  : [2] com.google.android.finsky.scheduler.k.a(69): ConstraintMapping: 26-1414141414,  -> L: 31756807ms, D: 32656807ms, C: false, I: false, N: 0
01-18 13:22:04.174 11343 11343 I Finsky  : [2] com.google.android.finsky.scheduler.JobSchedulerEngine.a(109): Cancelling existing job with id: 9003
01-18 13:22:04.178 11343 11343 I Finsky  : [2] com.google.android.finsky.scheduler.JobSchedulerEngine.a(48): Scheduling job Id: 9001, L: 2955561, D: 40924335, C: false, I: false, N: 1
01-18 13:22:04.182 11343 11343 I Finsky  : [2] com.google.android.finsky.scheduler.JobSchedulerEngine.a(48): Scheduling job Id: 9002, L: 31756807, D: 32656807, C: false, I: false, N: 0
```

**Comment by Serhy** [comment]
Same with release Android 7 (Samsung J7) - account. `Creating your account...` screen with spinner stays for 2+ minutes before Home page appears.

**Comment by mandrigin** [comment]
status-go has one commit that breaks it, right.
https://github.com/status-im/status-go/tree/release/0.19 should be a base for this status-go build

<blockquote><div> GitHub</div><div><strong><a href="https://github.com/status-im/status-go">status-im/status-go</a></strong></div><div>The Status module that consumes go-ethereum. Contribute to status-im/status-go development by creating an account on GitHub.</div></blockquote>

**Comment by mandrigin** [comment]
@annadanchenko thanks a lot for this log, it helped a ton!

**Comment by mandrigin** [comment]
@PombeirP @annadanchenko I will build a new release now, update this PR and it should fix the issue

**Comment by mandrigin** [comment]
@annadanchenko @PombeirP @antdanchenko updated, lets see how it works now

**Comment by annadanchenko** [comment]
jenkins/prs/ios — This commit cannot be built @mandrigin @jakubgs

**Comment by mandrigin** [comment]
@annadanchenko my mistake, sorry

**Comment by mandrigin** [comment]
it is fixed now

**Comment by cammellos** [comment]
I have another bugfix , could you please hold on merging in case the test is ok? https://github.com/status-im/status-go/pull/1354/files , sorry for the inconvenience

<blockquote><div> GitHub</div><div><strong><a href="https://github.com/status-im/status-go/pull/1354">Fix broken migration by cammellos · Pull Request #1354 · status-im/status-go</a></strong></div><div>The migration from unencrypted-&gt;encrypted did not work properly, it was
deleting and re-creating the database. This correctly migrates the db.
I really thought I tested it and was fine, but I mu...</div></blockquote>

**Comment by annadanchenko** [comment]
on iOS build 4 and Android build 3 can create account
testing is blocked on Windows because same https://github.com/status-im/status-react/issues/7299 happens here

**Comment by annadanchenko** [comment]
> I have another bugfix , could you please hold on merging in case the test is ok?

@cammellos do you mean if test with creating account is ok or if we complete testing of this PR and find no more issues?

**Comment by annadanchenko** [comment]
@PombeirP currently, testing on all desktop builds are blocked by #7299 we will continue testing on mobile but need new build when #7299 will be fixed to recheck on desktop

**Comment by cammellos** [comment]
@annadanchenko if you complete testing for this PR and find no issues, I would like to add a commit for status-go on top (it's a small fix), fixing the ios issue

**Comment by Serhy** [comment]
There is no Group Chat visible on iOS device using who upgraded from 0.9.32 build on to this PR build (https://i.diawi.com/mb6d7z) and user who uses 0.9.32 Release version. 

### Steps to reproduce:

1) On iOS (12.1.2 in my case) and Android install release 0.9.32 app version and create new accounts on both devices (`UseriOS` and `UserAndroid`)
2) UseriOS adds UserAndroid in contacts
3) UseriOS creates Group Chat with UserAndroid (e.g. with title `GroupChatiOS`)
4) UserAndroid creates Group Chat with UseriOS (e.g. with title `GroupChatAndroid`)
5) UseriOS upgrades to build made off this PR (https://i.diawi.com/mb6d7z)
6) USeriOS logs in, navigates to GroupChats and taps JOIN CHAT. UseriOS sends messages in those Group Chats. UserAndroid (who still on 0.9.32 version also sends messages)

**Actual result:** None of messages are received by both iOS and Android user. They can't communicate in GroupChats. If to create new Group Chat by any of iOS and Android - new GroupChat also won't appear on Home screen on any side.

<blockquote><div><strong><a href="https://i.diawi.com/mb6d7z">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Comment by cammellos** [comment]
@Serhy this should be ready to be tested again, once the build is ready, thanks!

**Comment by annadanchenko** [comment]
@cammellos @jakubgs Android builds failed:
https://ci.status.im/job/status-react/job/prs/job/android/job/PR-7294/5/console

```
09:36:28 FAILURE: Build failed with an exception.
09:36:28 
09:36:28 * What went wrong:
09:36:28 A problem occurred configuring project ':app'.
09:36:28 > You have not accepted the license agreements of the following SDK components:
09:36:28   [Android SDK Platform-Tools].
09:36:28   Before building your project, you need to accept the license agreements and complete the installation of the missing components using the Android Studio SDK Manager.
```

**Comment by pedropombeiro** [comment]
@cammellos I rebased and force-pushed with an additional commit from Jakub which I believe will fix this issue.

**Comment by pedropombeiro** [comment]
I'm currently rebuilding the Docker image and will kick a new build once it's pushed.

**Comment by annadanchenko** [comment]
@PombeirP @cammellos  if provide wrong password then no error is shown and Signing you in is shown forever
Android phone, build 11
logcat:  [wrongpass.txt](https://github.com/status-im/status-react/files/2778477/wrongpass.txt)

same scenario works fine on last nightly

**Comment by cammellos** [comment]
@annadanchenko should be fixed in the new builds, thanks!

**Comment by annadanchenko** [comment]
@cammellos or @PombeirP can it be rebased, please? so new infura project id and other changes are included

**Comment by annadanchenko** [comment]
@cammellos @PombeirP on Windows last build 13 I can't create account - if provide password then creating account never ends. there is no logs saved. 

note that on Android and iOS I can create accounts fine. On today's nightly Windows build account can be created fine too.

**Comment by pedropombeiro** [comment]
@annadanchenko is it just Windows or Desktop as a whole?

**Comment by cammellos** [comment]
@PombeirP desktop, I just tried on linux, I think a commit has been lost while force pushing

**Comment by churik** [comment]
Same for Mac OS. 
Log: [Status.log](https://github.com/status-im/status-react/files/2787946/Status.log)

**Comment by cammellos** [comment]
It's a config change, `BackupDisabledDataDir` was before defaulted in status-go to `DataDir`, now it's not defaulted anymore, so I will explicitly pass it.

File: src/status_im/utils/platform.cljs [hunk 1]
```diff
@@ -28,8 +28,7 @@
 (def platform-specific
   (cond
     android? android/platform-specific
-    ios? ios/platform-specific
-    :else desktop/platform-specific))
+    ios? ios/platform-specific))
```
Comment (mandrigin) [inline src/status_im/utils/platform.cljs]: I don't understand what this change does...

Comment (cammellos) [reply src/status_im/utils/platform.cljs]: ouch changed wrong line!

Comment (cammellos) [reply src/status_im/utils/platform.cljs]: done, mistakenly commited that change

**Comment by cammellos** [comment]
we are having some jenkins trouble, can't build, @jakubgs is looking into it

**Comment by churik** [comment]
@cammellos tested https://ci.status.im/blue/organizations/jenkins/status-react%2Fprs%2Fmacos/detail/PR-7294/15/artifacts build and got stuck in creating  account after set up a password.
Log: [Status.log](https://github.com/status-im/status-react/files/2788293/Status.log)

**Comment by cammellos** [comment]
@churik thanks, rebuilding now, I will test it and let you know

**Comment by cammellos** [comment]
@annadanchenko it should be fixed in the last build

**Comment by annadanchenko** [comment]
> @annadanchenko it should be fixed in the last build

thanks @cammellos indeed account can be created fine on new Windows build https://ci.status.im/job/status-react/job/prs/job/windows/job/PR-7294/20/

**Comment by pedropombeiro** [comment]
OK, Linux build is finally green.

**Comment by annadanchenko** [comment]
Update: can reproduce in nightly, https://github.com/status-im/status-react/issues/7332
@cammellos empty error popup is shown after sending SNT to a contact from Wallet.
screenshot_20190124-110210

I'm testing on recovered account, real Android 9 device, build 25. Can also reproduce on fresh account
```
01-24 11:01:24.686 27506 27616 I ReactNativeJS: ***
01-24 11:01:24.686 27506 27616 I ReactNativeJS: NAME: nil
01-24 11:01:24.686 27506 27616 I ReactNativeJS: MESSAGE: "Invalid push notification payload{:from \"0x043a5583b23ed88ef030fe4636c8cf7d5bace61f928a626726dacc9f1ece90012f5cfc7a141d883a6b86254f743fc52c8343496c28152cbe2aad920b7fbaa6e2bd\", :to nil, :id \"0x6b4e146b37eab0662ce9db71ef9864b0bf657fe3497c3197cc7158349b357394\"}"
01-24 11:01:24.686 27506 27616 I ReactNativeJS: 
01-24 11:01:24.686 27506 27616 I ReactNativeJS: 
01-24 11:01:24.686 27506 27616 I ReactNativeJS: 
01-24 11:01:24.686 27506 27616 I ReactNativeJS: ***
```

Steps:
1. create account, add a contact, get some ETH and SNT
2. join #status and #testanna chats
3. relaunch app
4. open Wallet and send to a contact 0.08 SNT - on transaction sent screen wait for 20-30 seconds

[Status.log](https://github.com/status-im/status-react/files/2791142/Status.log)
[geth.log](https://github.com/status-im/status-react/files/2791145/geth.log)

**Comment by cammellos** [comment]
@annadanchenko @goranjovic had the same issue possibly on develop yesterday, @goranjovic could you confirm?

**Comment by annadanchenko** [comment]
> @annadanchenko @goranjovic had the same issue possibly on develop yesterday, @goranjovic could you confirm?

yes, it's in nightly too https://github.com/status-im/status-react/issues/7332

**Comment by churik** [comment]
Tested Mac OS and Linux builds - fresh install, upgrade from current nightly, creating new account, restoring account from seed phrase, everything looks fine

**Comment by annadanchenko** [comment]
on iOS, see no checkbox next to device to pair with. If click on the line next to device name that "invisible" checkbox is set to ON and becomes visible

**Pull request merged**